packages/esm-reports-app/src/components/reports.resource.tsx

```diff
@@ -0,0 +1,229 @@
+import { openmrsFetch, Session, FetchResponse, openmrsObservableFetch } from '@openmrs/esm-framework';
```

Comment:
Annotate type imports with `type`:

```suggestion
import { openmrsFetch, Session, type FetchResponse, openmrsObservableFetch } from '@openmrs/esm-framework';
```

Reply:
fixed